.github/workflows/dep_compat.yaml

```diff
@@ -27,5 +30,6 @@ jobs:
           SUI_NODE_ACCESS_KEY: ${{ secrets.SUI_SUPER_ACCESS_KEY }}
           SUI_GAS_ACCESS_KEY: ${{ secrets.SUI_SUPER_ACCESS_KEY }}
           SUI_WALLET_ACCESS_KEY: ${{ secrets.SUI_SUPER_ACCESS_KEY }}
+          APTOS_NODE_ACCESS_KEY: ${{ secrets.APTOS_SUPER_ACCESS_KEY }}
```

Comment:
This was missed from https://github.com/shinamicorp/shinami-typescript-sdk/pull/37